eo-maven-plugin/pom.xml

```diff
@@ -169,13 +169,11 @@ SOFTWARE.
       <groupId>javax.json</groupId>
       <artifactId>javax.json-api</artifactId>
       <version>1.1.4</version>
-      <scope>runtime</scope>
```

Comment:
@volodya-lombrozo maybe it's better to replace with `provided`? https://howtodoinjava.com/maven/maven-dependency-scopes/